search-sentiments-map/test/search-test.js

```diff
@@ -1,35 +1,220 @@
 const chai = require('chai');
-const should = require('chai').should();
-const expect = require('chai').expect;
+const assert = chai.assert;
 const sinon = require('sinon');
 const search = require('./../routes/search').search;
+const searchInterestsModule = require('./../routes/search-interests.js');
+const sentiment = require('./../routes/sentiment.js');
 const {Datastore} = require('@google-cloud/datastore');
+const datastore = new Datastore();
+const fetch = require('node-fetch'); // Used to access custom search.
 
 describe('Search', function() {
+  describe('RetrieveSearchResultFromDatastore', function() {
+    let mockData;
+    beforeEach(() => {
+      mockData = {
+        topic: 'testTopic',
+        timestamp: 0,
+        dataByCountry: 'dataByCountry',
+      };
+
+      // Calls to the datastore are stubbed.
```

Comment:
Prefer active voice over passive.

```suggestion
      // Stub calls to the datastore.
```